projects/packages/wp-js-data-sync/src/schema/modifiers/class-decorate-with-default.php

```diff
@@ -18,15 +18,30 @@ public function __construct( Parser $parser, $default_value ) {
 		$this->default_value = $default_value;
 	}
 
-	public function parse( $input_value ) {
+	public function parse( $value, $meta = null ) {
 		try {
-			return $this->parser->parse( $input_value );
-		} catch ( \Error $e ) {
+			return $this->parser->parse( $value, $meta );
+		} catch ( \Exception $e ) {
 			return $this->default_value;
 		}
 	}
 
 	public function get_default_value() {
 		return $this->default_value;
 	}
+
+	public function __toString() {
+		return $this->parser->__toString() . ' (default: ' . $this->default_value . ')';
+	}
+
+	#[\ReturnTypeWillChange]
```

Comment:
https://php.watch/versions/8.1/ReturnTypeWillChange

Does this belong above or in the function? I'm new to these tags.

Reply:
Right above the function definition